.github/workflows/pr-labels.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  pull_request:
+   types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  check_pr_labels:
+    runs-on: ubuntu-latest
+    name: Verify that the PR has a valid label
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Verify PR label action
+        uses: ./
+        id: verify-pr-label
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          valid-labels: 'release:api-changes, releasex:testing, release:documentation, release:refactoring, release:features, release:bugfixes, release:release, release:documentation'
```

Comment:
`release:release` is not a label, as described in Slack. Instead they're classified as documentation.

Reply:
Also, `documentation` shows up twice, and typo in `releasex:testing`.